scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -2053,14 +2209,15 @@ def run( self ):
 
                 runtime = None
                 if k8s_cache:
+                    # TODO(czerwin):  If the api server has been disabled, we need to decide what to do here.
```

Comment:
This TODO still needs to be handled.

Reply:
`k8s_cache` is always non-None now.  That being said getting the runtime can return `None` in which case metrics will be disabled.

It's probably ok to leave this as is with the merge to master (because it will still be calling out to the api).  It's only when the cache warmer stuff hits that it might be a problem and we can address it in a commit on that branch.